numpy/core/include/numpy/npy_3kcompat.h

```diff
@@ -325,7 +325,7 @@ PyObject_Cmp(PyObject *i1, PyObject *i2, int *cmp)
 {
     int v;
     v = PyObject_RichCompareBool(i1, i2, Py_LT);
-    if (v == 0) {
+    if (v == 1) {
```

Comment:
There are two more instances of this mistake, right after the next two calls to `PyObject_RichCompareBool` below.